### PR TITLE
fix: override the default table name and column name

### PR DESCRIPTION
**Pull request by RobotHuang**
Fix: https://github.com/casbin-rs/diesel-adapter/issues/17

Signed-off-by: RobotHuang <1183598761@qq.com>

### CODE REVIEW

**Comment by hsluoyz** [comment]
@hackerchai plz review.

/cc @GopherJ @xcaptain

**Comment by hsluoyz** [comment]
@RobotHuang plz fix CI:

image

**Comment by GopherJ** [comment]
actually I prefer name the table after casbin_rules than casbin_rule. 

I know that there are some js adapters using casbin_rule as table name but I feel it's a mistake there... because table has more than one row and cabin has more than one rule...

**Comment by hsluoyz** [comment]
> actually I prefer naming the table after casbin_rules than casbin_rule.
> 
> I know that there are some js adapters using casbin_rule as table name but I feel it's a mistake there... because table has more than one row and cabin has more than one rule...

I think it's OK because:

1. We usually say User table, Role table. Actually User table has multiple rows. So it's similar for casbin_rule.
2. There already dozens of existing adapters in different languages that have followed the name: casbin_rule. It's big work if we ever want to change it.

**Comment by xcaptain** [comment]
Just changed some names right? I didn't see code that map table names or column names. I'm ok to use `casbin_rule` over `casbin_rules`, but some existing users may be affected, so you'd better think of a more compatible solution.

File: src/models.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-use super::schema::casbin_rules;
+use super::schema::casbin_rule;
 
 #[derive(Queryable, Identifiable)]
 pub(crate) struct CasbinRule {
```
Comment (RobotHuang) [inline src/models.rs:4]: `failed to resolve: use of undeclared crate or module `casbin_rules``
I don't know how to solve this problem.

**Review by hackerchai — COMMENTED** [review]
Actually, I prefer `casbin_rules` rather than `casbin_rule`
But can you plz fix the CI error?

**Comment by hsluoyz** [comment]
Actually I'm personally OK for either `casbin_rules` or `casbin_rule`. But in order to keep compatibility between all Casbin projects, plz make PRs to fix all occurences of `casbin_rule` to `casbin_rules` for our repos.

**Comment by hackerchai** [comment]
@RobotHuang After the discussion with casbin community, we decide to use:
1. `casbin_rules` instead of `casbin_rule`
2. `ptype` instead of `p_type`

Thanks for your hard work, and you are always welcomed to contribute to casbin-rs. This PR will be closed.

**Comment by hsluoyz** [comment]
@hackerchai I think we still need to change the table name from `casbin_rules` to `casbin_rule`. So maybe we need to keep this PR open?

**Pull request closed without merging**